src/workloads/execution/MultiDeletesDocByDoc.yml

```diff
@@ -0,0 +1,27 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: >
+  Runs a workload that inserts and then deletes a large range of documents on one collection while
+  performing writes on another collection in the same database. Serves as a base line for comparing
+  legacy document by document deletes versus efficiently batched multi deletes.
+
+# Increase the maxPoolSize to enable faster initial data population
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 300
+      # Allow for longer duration since the loading of the data may take a while.
+      socketTimeoutMS: 3_600_000  # = 1 hour
+
+LoadConfig:
+  Path: "../../phases/execution/MultiDeletes.yml"
+  Parameters:
+    DocumentSizePaddingBytes: 1024 #1KB
+    DocumentCount: 5_000_000
```

Comment:
7 minutes for doc by doc deletions with concurrent writes in initial patch testing